src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.OSX/X509Pal.cs

```diff
@@ -31,15 +33,27 @@ private partial class AppleX509Pal : ManagedX509ExtensionProcessor, IX509Pal
                     switch (oid.Value)
                     {
                         case Oids.RsaRsa:
+                            if (key.IsInvalid)
+                            {
+                                throw Interop.AppleCrypto.CreateExceptionForOSStatus(errSecUnsupportedKeyFormat);
```

Comment:
Is this for symmetry, or is it actually hit?  If it's just symmetry I'd rather not have it.

Reply:
Just symmetry.